README.md

```diff
@@ -1 +1,26 @@
-# threat-intelligence
\ No newline at end of file
+# threat-intelligence
+
+Infoblox's [Threat Intelligence Group (TIG)](https://www.infoblox.com/cyber-threat-intelligence/) detects, curates, and
+publishes threat intelligence data pertaining to relevant cyber campaigns. While majority of Infoblox's threat intelligence
+is disclosed [privately](https://www.infoblox.com/products/threat-intelligence/) to paying customers, TIG shares 
+indicators of compromise (IOCs) related to threats that are of high interest to the cyber security community through
+this public repository. The following contains descriptions about the contents of each dataset (i.e data folder). All files
+are csv formatted and [MISP](https://www.misp-project.org/) compatible.
+
+
+### ukraine
+This folder contains IOCs related to the Russia-Ukraine conflict. Majority of the content has been organically generated
+using both internal and private data. The `ukraine_russia_legitimate_iocs.csv` file contains confirmed IOCs that are
+managed by legitimate organizations. This includes domains belonging to well-known donation sites providing support to
```

Comment:
I would say legitimate file "contains confirmed indicators that, at the time of review, were not evidently associated with malicious activity. The related websites did not show indications of hosted malware or fraudulent behavior, but may still host content that is not wanted by some users."   (we have dating sites, e.g., in there)